packages/catalog-realm/fields-preview/geo-search-point.gts

```diff
@@ -0,0 +1,52 @@
+import { GeoSearchPointField } from '../fields/geo-search-point';
+
+import {
+  CardDef,
+  field,
+  contains,
+  type BaseDefConstructor,
+  type Field,
+} from 'https://cardstack.com/base/card-api';
+import { Component } from 'https://cardstack.com/base/card-api';
+import { FieldContainer } from '@cardstack/boxel-ui/components';
+import { getField } from '@cardstack/runtime-common';
+
+export class GeoSearchPointPreview extends CardDef {
+  @field geoSearchPoint = contains(GeoSearchPointField);
+
+  static displayName = 'Geo Search Point Preview';
+  static isolated = class Isolated extends Component<typeof this> {
+    <template>
+      <section class='fields'>
+        <FieldContainer
+          @label='Geo Search Point'
+          @icon={{this.getFieldIcon 'geoSearchPoint'}}
+        >
+          <FieldContainer @vertical={{true}} @label='Atom'>
+            <@fields.geoSearchPoint @format='atom' />
+          </FieldContainer>
+          <FieldContainer @vertical={{true}} @label='Embedded'>
+            <@fields.geoSearchPoint @format='embedded' />
+          </FieldContainer>
+        </FieldContainer>
+      </section>
+
+      <style scoped>
+        .fields {
+          display: grid;
+          gap: var(--boxel-sp-lg);
+          padding: var(--boxel-sp-xl);
+        }
+      </style>
+    </template>
+
```

Comment:
apart from getCardComponent. This is another one we always use. We shud find a way to clean this up. TBD